docs/_data/portland-2025-schedule.yaml

```diff
@@ -0,0 +1,156 @@
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '13:45'
+    title: 'Afternoon Session - all day groups reconvene and new group introductions'
+  - time: '16:30'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '17:00'
+    title: '<b>End of Writing Day</b>'
+  - time: '17:00'
+    title: '<b>Reception starts</b>'
+  - time: '19:00'
+    title: '<b>Reception ends</b>'
+
+talks_day1:
+  - time: '8:00'
+    title: Doors open
+  - time: '8:15'
+    title: '<i>Welcome Wagon conference introduction and venue tour</i>'
+  - time: '8:30'
+    title: '<b>Sponsor booths open</b>'
+  - time: '9:00'
+    duration: '0:25'
+    title: Write the Docs team - Conference introduction
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:00'
+    title: "<b>Unconference starts</b>"
+  - duration: '0:30'
+    slug: quest-for-the-holy-grail-turning-user-feedback-into-meaningful-change-rachel-rigdon
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: the-power-of-two-how-support-and-documentation-teams-transform-technical-conten-yanjie-niu
+  - duration: '0:5'
+    title: 'Sponsor intros'
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:15'
+    title: '☕ Coffee break'
+  - duration: '0:30'
+    slug: regulate-this-writing-for-the-medical-device-industry-holly-mickelson
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '1:20'
+    title: '🍽️ Lunch Break (80 mins)'
+  - duration: '0:30'
+    title: Lightning Talks (30 mins)
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: writing-the-shipwreck-stephanie-fuller
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: seven-habits-of-increasingly-technical-technical-writers-janine-chan
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:15'
+    title: '☕ Coffee break'
+  - duration: '0:30'
+    slug: embracing-the-kraken-how-to-build-a-docs-toolchain-monster-and-why-it-s-ok-ravind-kumar
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: docs-as-tests-a-strategy-for-resilient-docs-manny-silva
+  - duration: '0:10'
+    title: '✋ 10 min Q&A'
+  - duration: '0:10'
+    title: '<b>Group Photo! Right after the Q&A ;-)</b>'
+  - duration: '0:00'
+    title: '<b>Conference day 1 ends</b>'
+  - time: '18:30'
+    title: "<b>Social Event</b>"
+  - time: '20:30'
+    title: "<b>Social Event Ends</b>"
+
+talks_day2:
+  - time: '8:00'
+    title: Doors open
+    duration: '0:30'
+  - time: '8:30'
+    title: '<b>Sponsor booths open</b>'
+    duration: '0:30'
+  - duration: '0:00'
+    title: "<b>Unconference Starts</b>"
+  - duration: '0:30'
+    slug: docs-ai-tooling-is-better-and-better-for-us-than-you-think-alex-garnett
```

Comment:
I wonder if it makes sense to try and start the first talk 10 mins later or something to get a few more folks in seats? I feel like first thing Tuesday is always a hard slot, but probably too much hassle to adjust it?

Reply:
I feel like it would be better to push the whole thing by 30 minutes rather than 10? And with the 4pm finish we could do that. Then again, if folks have booked travel that's not fun.

So I'd vote for leave as is, and maybe bump later next year?